script.js

```diff
@@ -37,3 +38,130 @@ open.addEventListener('click', () => {
+    sourceButton: 'See Source',
+    live: 'images/live-version2.svg',
+    source: 'images/live-version.svg',
+  },
+  {
+    title: 'Uber Navigation',
+    devs: ['Uber', '•', 'Lead Developer', '•', '2018'],
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent, Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.',
+    tags: ['html', 'Ruby on rails', 'css', 'javascript'],
+    image: './images/Snapshoot-Portfolio4.png',
+    liveButton: 'See Live',
+    sourceButton: 'See Source',
+    live: 'images/live-version2.svg',
+    source: 'images/live-version.svg',
+  },
+];
+
+function display(index) {
+  const main = document.querySelector('main');
+  const {
+    title,
+    devs,
+    description,
+    tags,
+    image,
+    liveButton,
+    sourceButton,
+    live,
+    source,
+  } = projects[index];
+  const dev1 = devs[0];
+  const dev2 = devs[1];
+  const dev3 = devs[2];
+  const dev4 = devs[3];
+  const dev5 = devs[4];
+  const tag1 = tags[0];
+  const tag2 = tags[1];
+  const tag3 = tags[2];
+  const tag4 = tags[3];
+
+  const projectPopup = document.createElement('div');
+  projectPopup.className = 'project-popup';
+  projectPopup.innerHTML = `
+    <div class="desktop-window">
+    <div class="all-popup">
+        <div class="desktop-top-popup">
+          <h2>${title}</h2>
+          <button type='button' class="close-button">X</button>
+          </div>
+        <ul class="tech">
+        <li class="dev1">${dev1}</li>
+        <li class="dev2">${dev2}</li>
+        <li class="dev3">${dev3}</li>
+        <li class="dev4">${dev4}</li>
+        <li class="dev5">${dev5}</li>
+        </ul>
+        <img class="desktop-popup-snapshot" src=${image} alt="project 1 snapshot">
+        <div class="bottom-container">
+          <p class="p-popup">${description}</p>
+          <div class="small-container">
+            <ul class="prog-languages">
+              <li class="lang">${tag1}</li>
+              <li class="lang"  style="min-width: fit-content;">${tag2}</li>
+              <li class="lang">${tag3}</li>
+              <li class="lang">${tag4}</li>
+            </ul>
+            <hr class="line">
+            <section class="buttons">
+            <button type='button' class="popup-button">
+              ${liveButton}
+              <img class="button-logo" src=${live}>
+            </button>
+            <button type='button' class="popup-button">
+              ${sourceButton}
+              <img class="button-logo" src=${source}>
```

Comment:
- [ ] [Optional] Your live and source buttons need to be links so that when we press the buttons, we get taken to the website or repository for that specific project. I've added this as optional as the previous reviewer missed it but kindly do try to implement it.